source/server/hot_restart_impl.h

```diff
@@ -129,6 +132,7 @@ class HotRestartImpl : public HotRestart {
   SharedMemory* shmem_;
   ProcessSharedMutex log_lock_;
   ProcessSharedMutex access_log_lock_;
+  friend class HotRestartUdpForwardingTestHelper;
```

Comment:
nit: friend declarations should come at the top of the section

https://google.github.io/styleguide/cppguide.html#Declaration_Order

Reply:
Is it envoy policy to obey all google style rules?
I would expect this one would be enforced by a linter if we really want it enforced, but okay, done anyway.

Reply:
Yes, that's the policy:

Beyond code formatting, for the most part Envoy uses the Google C++ style guidelines. The following section covers the major areas where we deviate from the Google guidelines.

https://github.com/envoyproxy/envoy/blob/main/STYLE.md

The Style Guide go well beyond things which can be enforces with the tooling we have, annoyingly.

Reply:
Did you push a new commit? I'm not seeing a change, fwiw.

Reply:
Sorry about that, I had done the relevant `git push` but replied too quickly, before the git hooks had finished!